lib/install/vchlog/buffered_pipe.go

```diff
@@ -37,24 +41,28 @@ func NewBufferedPipe() *BufferedPipe {
 	var m sync.Mutex
 	c := sync.NewCond(&m)
 	return &BufferedPipe{
-		buffer: bytes.NewBuffer(nil),
-		c:      c,
-		closed: false,
+		buffer:      bytes.NewBuffer(nil),
+		c:           c,
+		readClosed:  false,
```

Comment:
The zero-value for booleans is `false`, so if you leave `readClosed`, `writeClosed` and `readerReader` out of this instantiation, they will init to `false` automatically. Feel free to leave it like this if you are doing it to be explicit/verbose!

Reply:
Thanks! I just feel like it's more explicit this way, to let code reader know that the pipe is default open and we're waiting for a reader to ready